src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInWidgets.ts

```diff
@@ -1,16 +1,16 @@
 import {
   ContentRecord,
   CubeName,
-  DataModel,
+  serializeWidgetState,
   deserializeWidgetState,
   isWidgetWithQueryState,
   MdxSelect,
   WidgetWithQueryState,
 } from "@activeviam/activeui-sdk-5.0";
+import { DataModel } from "@activeviam/activeui-sdk-5.1";
 import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
 import { produce } from "immer";
 import _mapValues from "lodash/mapValues";
-import { serializeWidgetState } from "@activeviam/activeui-sdk-5.1";
```

Comment:
@antoinetissier I thought we decided [here](https://github.com/activeviam/activeui-migration/pull/49#discussion_r1029280504) in #49  to call `deserializeWidgetState` from 5.0 and `serializeWidgetState` from 5.1, what is the reason for this change ?